### PR TITLE
Week 4 project - Sofie Pellegrini & Jessica Sandler

### PR DESCRIPTION
**Pull request by SofiePellegrini**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,47 +1,131 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
-
-// Global variables, if you need any, declared here
-
-// Functions declared here
+const inputWrapper = document.getElementById('input-wrapper')
+const form = document.getElementById('name-form')
+const nameInput = document.getElementById('name-input')
+const messageAudio = new Audio ("assets/click.wav")
```
Comment (nadialefebvre) [inline code/script.js:6]: Tiny writing comment: we were told that both simple and double quotation marks are fine to use in JS but to stick to one or the other :)

File: code/script.js [hunk 2]
```diff
@@ -1,47 +1,131 @@
...
+  } else {
+    showMessage(`My name is ${name}`, 'user')
+  }
+  nameInput.value = ''
+  showDestinations(name)
+}
+
+// The bot asks the user where he/she wants to travel
+const showDestinations = (name) => {
+  setTimeout(() => showMessage (`Hello ${name}!`, 'bot'), 1000)
```
Comment (nadialefebvre) [inline code/script.js:61]: Great with the delayed messages, it gives the user some sense of "normality", as if the bot was really typing one message after the other.

File: code/script.js [hunk 3]
```diff
@@ -1,47 +1,131 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
-
-// Global variables, if you need any, declared here
-
-// Functions declared here
+const inputWrapper = document.getElementById('input-wrapper')
```
Comment (nadialefebvre) [inline code/script.js:3]: Your variables are well named, it makes it easier to follow when you use them.

**Review by nadialefebvre — APPROVED** [review]
Jessica and Sofie, you did a great job! Your JavaScript is really clear and easy to follow, and your comments are really helpful so I understand what is what. To me it seems that you have done all the requirements and your chatbot works just fine. I can't give more comments since JS is totally new for me, but I don't see any faulty things. Good work!

File: code/script.js [hunk 4]
```diff
@@ -1,47 +1,131 @@
...
+  }
+  nameInput.value = ''
+  showDestinations(name)
+}
+
+// The bot asks the user where he/she wants to travel
+const showDestinations = (name) => {
+  setTimeout(() => showMessage (`Hello ${name}!`, 'bot'), 1000)
+  setTimeout(() => showMessage ('Where do you want to travel? ','bot'), 1300)
+  setTimeout(showDestinationButtons, 2000)
```
Comment (EmmaaBerg) [inline code/script.js:63]: I also liked that you have two different timeOut on the bot reply  and that the buttons appears after as well.

File: code/script.js (hunk 3 above)
Comment (EmmaaBerg) [reply code/script.js:3]: I agree, good with the `camelCase` for the variables and functions

File: code/script.js [hunk 5]
```diff
@@ -1,47 +1,131 @@
...
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hey you, What's your name?`, 'bot')
+  
+}
+// Function that saves user's name
+const handleNameInput = (event) => {
+  event.preventDefault()
+  let name = nameInput.value
+  // User name will be 'stranger' if there is no input
```
Comment (EmmaaBerg) [inline code/script.js:48]: Think this comment is helpful to understand the two outcomes of the user reply (just like the rest of your comments in the JS-file)

File: code/style.css [hunk 6]
```diff
@@ -120,15 +120,14 @@ main {
 }
 
 button {
-  background-color: #0026ff;
-  color: white;
+  background-color: rgb(38, 137, 230);
+  color: black;
```
Comment (EmmaaBerg) [inline code/style.css:124]: This is a small comment, but both hex and rgb are used for the colors. Maybe the ones where the color is typed out `color: black` alos could be named with hex just to try to do it more the same throughout the code?

**Review by EmmaaBerg — COMMENTED** [review]
I agree with Nadia, I think you have written a clean and structured code that reaches the blue and general requirements. The comments in the JS-file makes it easy to understand and follow and I get the feeling that you both understand what you have written in the code.

**Pull request closed without merging**